src/js/utils/events.js

```diff
@@ -127,8 +127,9 @@ export function off(elem, type, fn) {
  * Trigger an event for an element
  * @param  {Element|Object}      elem  Element to trigger an event on
  * @param  {Event|Object|String} event A string (the type) or an event object with a type attribute
+ * @param  {...*} [evtData] you may pass additional data with the event
  */
-export function trigger(elem, event) {
+export function trigger(elem, event, ...evtData) {
   // Fetches element data and a reference to the parent (for bubbling).
```

Comment:
Would it make sense to enforce the use of a data hash on Events.trigger as I did on component.trigger??